symbolic-debuginfo/src/pe.rs

```diff
@@ -229,6 +248,17 @@ impl<'data> PeObject<'data> {
         &self.pe.sections
     }
 
+    /// Returns the `SectionTable` for the section with this name, if present.
+    pub fn section(&self, name: &str) -> Option<SectionTable> {
```

Comment:
can we leave this as private, or does it need to be `pub`?

Reply:
I didn't end up using this function. Removed.